Examples/Scripts/MaterialMaping/configureMap.py

```diff
@@ -23,58 +23,67 @@
-                            if (config[Name]['approach'][dapp['stype']][kapp][0][2] != 1) or (config[Name]['approach'][dapp['stype']][kapp][1][2] != 1) :
-                                data['volumes'][kvol]['layers'][klay]['approach'][kapp]['mapMaterial'] = True
-
-
+                            data['volumes'][kvol]['layers'][klay]['approach'][kapp]['bin0']        = config[Name]['approach'][dapp['stype']][kapp]['bin0']
+                            data['volumes'][kvol]['layers'][klay]['approach'][kapp]['bin1']        = config[Name]['approach'][dapp['stype']][kapp]['bin1']
+                            data['volumes'][kvol]['layers'][klay]['approach'][kapp]['mapMaterial'] = config[Name]['approach'][dapp['stype']][kapp]['mapMaterial']
+                                
                     if 'sensitive' in data['volumes'][kvol]['layers'][klay] :
                         for ksen  in data['volumes'][kvol]['layers'][klay]['sensitive'] :
                             dsen = data['volumes'][kvol]['layers'][klay]['sensitive'][ksen]
-                            data['volumes'][kvol]['layers'][klay]['sensitive'][ksen]['bin0'] = config[Name]['sensitive'][dsen['stype']][0]
-                            data['volumes'][kvol]['layers'][klay]['sensitive'][ksen]['bin1'] = config[Name]['sensitive'][dsen['stype']][1]
-                            if (config[Name]['sensitive'][dsen['stype']][0][2] != 1) or (config[Name]['sensitive'][dsen['stype']][1][2] != 1) :
-                                data['volumes'][kvol]['layers'][klay]['sensitive'][ksen] = True
-
-    json_file.seek(0)
+                            data['volumes'][kvol]['layers'][klay]['sensitive'][ksen]['bin0']        = config[Name]['sensitive'][dsen['stype']]['bin0']
+                            data['volumes'][kvol]['layers'][klay]['sensitive'][ksen]['bin1']        = config[Name]['sensitive'][dsen['stype']]['bin1']
+                            data['volumes'][kvol]['layers'][klay]['sensitive'][ksen]['mapMaterial'] = config[Name]['sensitive'][dsen['stype']]['mapMaterial']
+
+                            
+            if 'material' in data['volumes'][kvol] :
+                data['volumes'][kvol]['material']['mapMaterial'] = config[Name]['material']['mapMaterial']
+                for bin in data['volumes'][kvol]['material'] :
+                    if bin == 'bin0' :
+                        data['volumes'][kvol]['material']['bin0'] =  config[Name]['material']['bin0']
+                    if bin == 'bin1' :
+                        data['volumes'][kvol]['material']['bin1'] =  config[Name]['material']['bin1']
+                    if bin == 'bin2' :                
+                        data['volumes'][kvol]['material']['bin2'] =  config[Name]['material']['bin2']
+
+                        
+    json_file.seek(0) 
     json.dump(data, json_file, indent=4)
-    json_file.truncate()
+    json_file.truncate()
```

Comment:
Newline at the end of file would be better.

Reply:
Done